hack/verify-flags-underscore.py

```diff
@@ -55,6 +55,8 @@ def get_all_files(rootdir):
         # don't visit certain dirs
         if 'vendor' in dirs:
             dirs.remove('vendor')
+        if 'staging' in dirs:
```

Comment:
This pr doesn't cause that. It's a check run by hack/verify-flags-underscore.py. Please follow the instruction in the error to add the flag to known-flags.txt